app/v1/controllers/BaseController.scala

```diff
@@ -32,6 +37,19 @@ trait BaseController {
       result.copy(header = result.header.copy(headers = result.header.headers ++ newHeaders))
     }
   }
+
+  protected def getCorrelationId(errorWrapper: ErrorWrapper)(implicit logger: Logger): String = {
+    errorWrapper.correlationId match {
+      case Some(correlationId) => logger.info(s"[${logger.underlyingLogger}] - " +
```

Comment:
Do we care that `logger.underlyingLogger.toString` will evaluate to e.g. `Logger[v1.controllers.CreateBFLossController]`? 

Could just use `logger.info(s"[${getClass.getSimpleName}]...` if we want it to look like it does currently.

and then there is no need to pass a logger (just declare one in the BaseController).

Reply:
Other than maybe that, looks good.

Reply:
I hadn't thought of that actually, its nicer than sending an implicit logger.